app/src/main/java/org/simple/clinic/patient/PatientSearchResult.kt

```diff
@@ -95,7 +95,9 @@ data class PatientSearchResult(
     val phoneUpdatedAt: Instant?,
 
     @Embedded(prefix = "lastSeen_")
-    val lastSeen: LastSeen?
+    val lastSeen: LastSeen?,
+
+    val isAtAssignedFacility: Boolean?
```

Comment:
It seems like this is nullable only because we cannot add this in the view, right? If this is the case, we should not be adding this here since it should never be null from a modelling perspective (a patient can either be assigned to the current facility or not).

Reply:
Hmm, but we cannot do the assigned facility check when creating the view since it's dynamic. So that's why I am setting these values in the query. Is there a way we can do this?

Reply:
Does this help?

https://github.com/simpledotorg/simple-android/pull/2222#discussion_r549331706

We can then leave the boolean flag only at the SQL level and then add the assigned facility ID in the search result as a way to test this condition.